### ISSUE TITLE
How to reduce learning rate based last epoch valid acc or valid loss?

### ISSUE THREAD

**Issue by nlpjoe** [keras-team/keras#11094]
Time-decay or Drop-decay maybe a good strategy to train a model, but I think a better way is to reduce the learning rate when valid loss increasing or valid acc decreasing.  But I can only change learning rate by passing a `epoch` parameter into LearningRateSchedule. How can I do it?

**Comment by nlpjoe** [comment]
I found `ReduceLROnPlateau`.
